bedrock/mozorg/urls.py

```diff
@@ -317,4 +317,7 @@
     url(r'^xbl$', views.namespaces, {'namespace': 'xbl'}),
 
     page('locales', 'mozorg/locales.html'),
+
+    # TODO remove before merging!
```

Comment:
Note to self: remove this test page before merging.

Reply:
😄 time to remove!